### ISSUE TITLE
Feature - gwtCodeServer task

### ISSUE THREAD

**Issue by confile** [Putnami/putnami-gradle-plugin#4]
A gwtSuperDev mode task. It would be great if you could add this to your plugin. The code server should run independently of an application server. This gives you more freedom of choice.

**Comment by fdumay** [comment]
Enhancement accepeted, 
What do think of naming this task gwtCodeServer rather than gwtSuperDev?

**Comment by confile** [comment]
ok done

**Comment by fdumay** [comment]
the task gwtCodeServer is available on the snapshot, the version 0.2 will be released soon.
You can try it with :

```
buildscript {
	repositories {
		mavenCentral()
		maven{ url 'https://oss.sonatype.org/content/repositories/snapshots/' }
	}
	dependencies {
		classpath 'fr.putnami.gwt:putnami-gradle-plugin:0.2.0-SNAPSHOT'
	}
}

apply plugin: 'fr.putnami.gwt'

repositories {
	mavenCentral()
	maven{ url 'https://oss.sonatype.org/content/repositories/snapshots/' }
}
```

**Comment by confile** [comment]
I will try it.
